docs/dev/setup/kubernetes.rst

```diff
@@ -0,0 +1,288 @@
+
+5. Open Rancher and update the password
+
+Open Rancher on `<https://rancher.localhost/>`__.
+
+You will be notified that the connection is not private. The reason for that is that the Rancher deployment uses a self-signed certificate by an unknown authority 'dynamiclistener-ca'. 
+It is used for secure communication between internal components. Since it's your local environment this is not an issue and you can proceed to the website.
+
+You will be prompted to set a password which later will be used to login to Rancher. The password will be used often, that's why you shouldn't forget it.
+
+.. figure:: kubernetes/rancher_password.png
+   :align: center
+
+Then you should save the Rancher Server URL, please use the predefined name.  
+
+.. figure:: kubernetes/rancher_url.png
+   :align: center
+
+After saving you will be redirected to the main page of Rancher, where you see your cluters. There will be one local cluster.
+
+.. figure:: kubernetes/rancher_cluster.png
+   :align: center
+
+You can open the workloads using the menu, there will be no workloads deployed at the moment.
+
+.. figure:: kubernetes/rancher_nav_workloads.png
+   :align: center
+
+
+.. figure:: kubernetes/rancher_empty_workloads.png
+   :align: center
+
+6. Create new namespace in Rancher
+
+Namespaces are virtual clusters backed by the same physical cluster. Namespaces provide a scope for names. Names of resources need to be unique within a namespace, but not across namespaces.
+Usually different namespaces are created to separate environments deployments i.e. development, staging, production.
+
+For our development purposes we will create a namespace called artemis.
+It can be done easily using Rancher.
+
+a. Navigate to Namespaces using the top menu of Rancher
+
+b. Select ``Add Namespace`` to open the form for namespace creation  
+
+   .. figure:: kubernetes/rancher_namespaces.png
+      :align: center
+
+c. Put ``artemis`` as namespace's name and select the ``Create`` button
+
+   .. figure:: kubernetes/rancher_create_namespace.png
+      :align: center
+
+
+
+Create DockerHub repository
+---------------------------
+The Artemis image will be stored and managed in DockerHub. Kubernetes will pull it from there and deploy it afterwards.
+After you log in to your `DockerHub <https://hub.docker.com/>`__ account you can create as many public repositories as you want.
+In order to create a repository you need to select the ``Create repository`` button.
+
+
+**DockerHub:**
+
+.. figure:: kubernetes/dockerhub.png
+   :align: center
+
+Then fill in the repository name with ``artemis``. The use the ``Create`` button and you repository is created.
+
+.. figure:: kubernetes/dockerhub_create_repository.png
+   :align: center
+
+Configure Docker ID (username)
+------------------------------
+The username in DockerHub is called Docker ID. You need to seet your Docker ID in the ``artemis-deployment.yml`` resource so that Kubernetes knows where to pull the image from.
```

Comment:
```suggestion
The username in DockerHub is called Docker ID. You need to set your Docker ID in the ``artemis-deployment.yml`` resource so that Kubernetes knows where to pull the image from.
```